app/forms/instance-create.tsx

```diff
@@ -68,8 +70,10 @@ export type InstanceCreateInput = Assign<
     disks: DiskTableItem[]
     bootDiskName: string
     bootDiskSize: number
-    bootDiskSourceType: 'disk' | 'image'
-    bootDiskSource: string
+    bootDiskSourceType: BootDiskSourceType
+    siloImageSource: string
+    projectImageSource: string
+    diskSource: string
```

Comment:
I'd put newlines around these 4 lines and a comment above saying `type` is a switch picking between the other three